### DISCUSSION TITLE
[Question] : strange message in PIO

### DISCUSSION THREAD

**Discussion by hasenradball** [Bodmer/TFT_eSPI#1749] [Q&A - General]
Hi I get an strange meassage when using PIO:

![grafik](https://user-images.githubusercontent.com/26808969/161433890-5a8b6adb-4ce3-4c92-b8aa-2bbe4a205bf8.png)

I newer got this meassage before.
Whats behind this meassage?

**Comment by Bodmer** [comment]
Latest library version is 2.4.43 so maybe upgrade and see if it goes away. Other than that suggestion I have no idea.
